test/pkg/environment/aws/environment.go

```diff
@@ -143,8 +143,6 @@ func GetTimeStreamAPI(session *session.Session) timestreamwriteiface.TimestreamW
 
 func (env *Environment) DefaultEC2NodeClass() *v1.EC2NodeClass {
 	nodeClass := test.EC2NodeClass()
-	// Set the AMI Family to AL2023 in case the AMISelectorTerms change.
-	nodeClass.Spec.AMIFamily = lo.ToPtr(v1.AMIFamilyAL2023)
```

Comment:
I'd assume that @njtran made this change because some of the tests that he added required this in order for the tests to succeed. Can we confirm which tests we need to add the AMIFamily to now